CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 lib_xcore_math change log
 =========================
 
+2.1.4
+-----
+
+* Fixes bug (issue #170) in `xs3_memcpy()`.
```

Comment:
Consider describing the actual issue resolved here such that users can use the change log as a quick reference as to if they should move to this version -  for example, "Fixed issue with xs3_memcopy when length is less than 16" or similar.